src/main/java/org/jitsi/videobridge/VideoChannel.java

```diff
@@ -876,7 +876,7 @@ public void run()
     public Endpoint getEndpoint()
     {
         EndpointBase endpointBase = super.getEndpoint();
-        if (endpointBase instanceof Endpoint)
+        if (endpointBase == null || endpointBase instanceof Endpoint)
```

Comment:
what's the scenario where this happens? (a videochannel doesn't have an endpoint) is there a time where we create a videochannel but don't set its endpoint yet?

Reply:
Jicofo always sets endpoints to channels, but that's not necessarily a requirement. I'll remove the whole methods as you suggested above.